command/format.go

```diff
@@ -401,6 +401,17 @@ func (t TableFormatter) OutputSecret(ui cli.Ui, secret *api.Secret) error {
 		for k, v := range secret.Auth.Metadata {
 			out = append(out, fmt.Sprintf("token_meta_%s %s %v", k, hopeDelim, v))
 		}
+
+		if secret.Auth.MFARequirement != nil {
```

Comment:
I am wondering if we should print the token related information when the MFA requirements is not nil (which means there is no token returned and the user needs to validate MFA)?

Reply:
I had the same thought, and it would be pretty easy to not print them. However, the conversation on Slack seemed to lean towards keeping them in: https://hashicorp.slack.com/archives/C025608TAR0/p1652446657666779

Reply:
Although I marked this line, my main point is about whether we want to print the token related information when we are following the two-phase login work flow. I think the slack thread is in favor of printing relevant information. 
I feel the same way as in maybe we should avoid printing the following fields when two-phase login workflow is executed:
`token`, `token_accessor`, `token_duration`, `token_renewable`, token_policies`, `identity_policies`, and `policies`. 

In fact, in two-phase login, none of the auth response information is returned to the login command. So, I think that information is not relevant. What do you think about that?

Reply:
I agree with you, the whole token section above the section I added could essentially be wrapped in an if/else as it's not relevant for 2FA. I'd be absolutely in favour of making the change (I think we are aligned) but the sense I got from the Slack thread was that this should be discussed a separate issue.

Would you like to leave a comment on the thread with your opinion? If there is alignment that this should be done in this PR I'd be more than happy to make the change!

Reply:
Updated as per discussion in sprint planning. Now, two-phase MFA response might look like this:
```
violethynes@violethynes-C02CW1WWMD6R Repositories % vault login -method userpass username=alice password=kQ3BlE+5M1libWJc
A login request was issued that is subject to MFA validation. Please make sure
to validate the login by sending another request to sys/mfa/validate endpoint.

WARNING! The VAULT_TOKEN environment variable is set! The value of this
variable will take precedence; if this is unwanted please unset VAULT_TOKEN or
update its value accordingly.

WARNING! The following warnings were returned from Vault:

  * A login request was issued that is subject to MFA validation. Please
  make sure to validate the login by sending another request to mfa/validate
  endpoint.

Key                                        Value
---                                        -----
mfa_request_id                             20424e2f-e963-b23d-00de-01f41209fbf9
mfa_constraint_l22_pingid_id               ac25e939-4b7e-41ec-db50-bc65da01a642
mfa_constraint_l22_pingid_uses_passcode    false
mfa_constraint_l22_totp_id                 34dade06-351a-e576-537e-0c7d5b0e6d84
mfa_constraint_l22_totp_uses_passcode      true
```

Reply:
This looks great!